### DISCUSSION TITLE
send me a email to create a domain

### DISCUSSION THREAD

**Discussion by avarmaa** [learndevopsonline/learndevopsonline#1578] [Q&A] [answered]
hello raghu,my email id is adprince15@gmail.com  can you please send me an email to create a domain as i have joined late and going through previous sessions now

**Comment by sreedharm07** [comment, marked as answer]
Hi @avarmaa 
https://youtu.be/VyzzEg7U7m0?si=jIAkzqlwQYillyjp
this is the viedio to create domain

**Reply by avarmaa** [reply]
Thanks
